### ISSUE TITLE
adopt `next/previous` in chat response accessible view

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#188188]
Useful so a screen reader user doesn't have to escape the view, navigate to the next element, and reopen the view.

This is possible now in notifications and should also be possible for notebook output cc @amunger
